README.md

```diff
@@ -23,9 +24,12 @@ visualize the data.
 
 ### Tutorials
 
-The notebooks in the folder `tutorials/` demonstrate how to use both
-the API and the interactive GUI. You can view these notebooks online
-[here](https://github.com/openPMD/openPMD-viewer/tree/master/tutorials).

```

Comment:
I would still keep the paragraph
```
The notebooks in the folder `tutorials/` demonstrate how to use both
the API and the interactive GUI. You can view these notebooks online
[here](https://github.com/openPMD/openPMD-viewer/tree/master/tutorials).
``` 
and only add the paragraph
```
Alternatively you can even [run this tutorials online](https://mybinder.org/v2/gh/openPMD/openPMD-viewer/master?filepath=tutorials%2F)!
```
The main reason is that, in my experience, Binder is often down, (see e.g. [here](https://github.com/binder-project/binder/issues/95)) and so the user should have an easy way to still see the tutorials if this happens.

Reply:
good point, updated the PR accordingly.